scripts/create-unbound.sh

```diff
@@ -52,14 +53,23 @@ while read entry; do
 					continue
 				fi
 				parsed=$(echo $fileentry | sed -e "s/^\*\.//")
+                # Ignore empty lines
+                if [[ -z "$parsed" ]]; then
+                    continue
+                fi
 				if grep -qx "$parsed" $outputfile; then
 					continue
 				fi
+                isvalid=true
 				echo "  local-zone: \"${parsed}\" redirect" >> $outputfile
 				for i in ${cacheip}; do
 					echo "  local-data: \"${parsed} 30 IN A ${i}\"" >> $outputfile
 				done
 			done <<< $(cat ${basedir}/$filename | sort);
+            # Delete files with no entries
+            if [[ $isvalid == false ]]; then
+                rm $outputfile
+            fi
```

Comment:
```suggestion
```

I think this logic can be simplified by only inserting "server:" if we have actually found matches.
So remove line 48 and add the following just before original line 58:
```bash
        if [[ $(head -n 1 $outputfile) != "server:" ]]; then
            echo "server:" >> $outputfile
        fi
```

I'll include a patch so you can in my review commentary too.